pkg/option/config.go

```diff
@@ -1881,6 +1889,16 @@ func (c *DaemonConfig) HealthCheckingEnabled() bool {
 	return c.EnableHealthChecking
 }
 
+// HealthCheckNodePortEnabled returns true if health checking for nodeport by cilium is enabled
+func (c *DaemonConfig) HealthCheckNodePortEnabled() bool {
+	return c.EnableHealthCheckNodePort
+}
+
+// GetKubeProxyReplacement returns the kubeproxyreplacement configured
+func (c *DaemonConfig) GetKubeProxyReplacement() string {
+	return c.KubeProxyReplacement
+}
+
```

Comment:
We don't tend do add getters to the DaemonConfig for simple booleans.

Reply:
Sure will remove it.